kuma/core/tests/test_locale_middleware.py

```diff
@@ -35,9 +35,14 @@ def test_specificity(self):
         self.assertRedirects(reponse, '/fr/', status_code=302)
 
     def test_partial_redirect(self):
-        """Ensure that /en/ gets directed to /en-US/."""
+        """
+        Ensure that /en/ gets directed to /en-US/.
+
+        This was handled in the LocaleURLMiddleware (302), but is now handled
```

Comment:
I do not like to handle this redirect with Redirect rule. As all the locale redirect is handled by  `LocaleURLMiddleware`, this `en` to `en-US` should also be handled by middleware.